webapp/Connector/src/store/Explorer.js

```diff
@@ -7,5 +7,200 @@ Ext.define('Connector.store.Explorer', {
 
     extend: 'LABKEY.app.store.OlapExplorer',
 
-    subjectName: 'Subject'
+    subjectName: 'Subject',
+
+    //Override: Mostly the same as OlapExplorer, see code inside //DataSpace override for actual overrides
+    loadDimension : function(selections, altRequestDimNamedFilters) {
```

Comment:
Yeah it was mostly to avoid having to triage. I prefer stub methods over event firing as the check of each each target needs to be synchronous. As for switching to hosting17.1, it's probably more overhead than to get triage on release, though it's safer..